include/Makefile.in

```diff
@@ -585,7 +585,7 @@ SUBDIRS = libmesh
 # cleaned by 'make distclean'
 DISTCLEANFILES = libmesh_config.h
 
-# Do not edit - automatically generated from rebuild_include_HEADERS.sh
+# Do not edit - automatically generated from ./rebuild_include_HEADERS.sh
```

Comment:
Ha!  Last time I re-bootstrapped I re-edited these to keep the diffs smaller.  But if I'm not the only one who [doesn't put . in their $PATH](https://unix.stackexchange.com/questions/65700/is-it-safe-to-add-to-my-path-how-come) then I say now is the time to rise up and take back our bootstrap output from the heathens!

Reply:
I knew someone who had a disastrous time recently because of the presence of `.` in his PATH. Yea I don't have it, and I never intend to. Haha yes now is the time to take it back!